tests/conftest.py

```diff
@@ -578,3 +584,57 @@ def variety_resources(
     resource.target = get_variety_target
     resource.class_labels = get_variety_classes_labels
     return resource
+
+
+@pytest.fixture
```

Comment:
is there a need to move these?  if not, it would save time for the reader to leave them closer to where the tests are.  maybe in `tests/drum/unit/conftest.py` if you want to move them out of the test itself?

Reply:
moved fixtures back